src/common/components/testimonial/testimonial.css

```diff
@@ -22,25 +22,22 @@
   }
 }
 
+.pullQuote>p>span {
+  color: #dd4814;
+  font-size: 2rem;
+  line-height: 0;
+  position: relative;
+  top: 0.7rem;
+  left: -0.1rem;
+}
+.pullQuote>p>span + span{
+  left: 5px;
+}
 .pullQuote>p {
   font-size:1em;
   font-style:normal;
-  margin:0
-}
-
-.pullQuote>p:first-of-type:before {
```

Comment:
Seems like there already are quote signs here in CSS, so it would be best to keep any needed changes (size, colour, position) in here instead of adding them in HTML and styling separately.